src/xrGame/GametaskManager.cpp

```diff
@@ -85,7 +86,11 @@ CGameTask* CGameTaskManager::HasGameTask(const TASK_ID& id, bool only_inprocess)
     FindTaskByID key(id, only_inprocess);
     auto it = std::find_if(GetGameTasks().begin(), GetGameTasks().end(), key);
     if (it != GetGameTasks().end())
+    {
+        g_DiscordRPC.Update(
+            FALSE, StringTable().translate((*it).game_task ? (*it).game_task->m_Title : "st_no_active_task").c_str());
```

Comment:
Since you declared first argument as bool, why do you use defines instead of standard bool variables, like `false` and `true`?